enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/ChartTypeSelector.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  ChartTypeSettings,
+  type UseChartTypeVisualizationsProps,
+  useChartTypeVisualizations,
+} from "metabase/query_builder/components/chart-type-selector";
+import { Stack } from "metabase/ui";
+
+import { useInteractiveQuestionContext } from "../context";
+
+const useChartTypeSelectorsInner = ({
```

Comment:
What's this for?
It's used only once and it wraps useChartTypeVisualizations without any change?

Reply:
that was useful when I first made this a month ago

Reply:
updated